DbgPack/asset1.py

```diff
@@ -0,0 +1,31 @@
+from dataclasses import dataclass, field
+from pathlib import Path
+
+from .abc import AbstractAsset
+from .struct_reader import BinaryStructReader
+
+
+@dataclass
+class Asset1(AbstractAsset):
```

Comment:
Don't forget a docstring, even if it's as simple as:
```suggestion
class Asset1(AbstractAsset):
    """
    Represents an asset entry in a pack1 file
    """
```